proxy/http3/Http3Transaction.cc

```diff
@@ -581,8 +585,12 @@ Http3Transaction::_process_write_vio()
 
   size_t nwritten = 0;
   bool all_done   = false;
-  this->_frame_collector.on_write_ready(this->_info.adapter.stream().id(), *this->_info.write_vio->get_writer(), nwritten,
-                                        all_done);
+  auto error      = this->_frame_collector.on_write_ready(this->_info.adapter.stream().id(), *this->_info.write_vio->get_writer(),
+                                                          nwritten, all_done);
+  if (error && error->cls != Http3ErrorClass::NONE) {
```

Comment:
What case does `error->cls` become `Http3ErrorClass::NONE` in?

Reply:
It was probably needed when we had NoError class. I think we can remove it now.

Reply:
Yeah, good point, I'll make the changes.

Reply:
I think it is because the `QUICErrorClass` uses a similar structure:
```
enum class QUICErrorClass {
  UNDEFINED,
  TRANSPORT,
  APPLICATION,
};
```

But in the docs I didn't see there are any other types for `Http3Errors`, so I think we can just remove the error class complete for `Http3Errors`

Reply:
Let's do that.